streamlit_pdf/frontend/src/PdfViewer.tsx

```diff
@@ -66,7 +67,9 @@ const PAGE_MARGIN = 12
  * @returns {ReactElement} The rendered PDF viewer component
  */
 function PDFViewer({ args, theme }: ComponentProps): ReactElement {
-  const { file, height = 600 } = args
+  const { file: baseFileUrl, height = 600 } = args
```

Comment:
nit: `baseFileUrl` sounds a bit more like it's the base URL for files. Maybe just `fileUrl`